packages/eslint-config-react-app/index.js

```diff
@@ -315,5 +341,37 @@ module.exports = {
     'flowtype/define-flow-type': 'warn',
     'flowtype/require-valid-file-annotation': 'warn',
     'flowtype/use-flow-type': 'warn',
+
+    // https://github.com/testing-library/eslint-plugin-jest-dom
+    ...(hasJestDom
```

Comment:
Is there harm in always including jest-dom rules even if using the node test environment?

Reply:
It should be fine 👍